gobblin-data-management/src/main/java/org/apache/gobblin/data/management/policy/SelectBetweenTimeBasedPolicy.java

```diff
@@ -94,17 +100,25 @@ public SelectBetweenTimeBasedPolicy(Optional<Period> minLookBackPeriod, Optional
       public boolean apply(TimestampedDatasetVersion version) {
         return version.getDateTime()
             .plus(SelectBetweenTimeBasedPolicy.this.maxLookBackPeriod.or(new Period(DateTime.now().getMillis())))
-            .isAfterNow()
-            && version.getDateTime().plus(SelectBetweenTimeBasedPolicy.this.minLookBackPeriod.or(new Period(0)))
-                .isBeforeNow();
+            .isAfterNow() && version.getDateTime()
+            .plus(SelectBetweenTimeBasedPolicy.this.minLookBackPeriod.or(new Period(0)))
+            .isBeforeNow();
       }
     };
   }
 
   protected static Period getLookBackPeriod(String lookbackTime) {
```

Comment:
Looks like this is just reformatting. Unless there is a reason to reformat, leave it as is.

Reply:
For better readability, prefer to have this reformatting.